soc/nordic/common/Kconfig

```diff
@@ -21,6 +21,13 @@ config NRF_SYS_EVENT
 	bool "nRF system event support"
 	select NRFX_POWER if !NRF_PLATFORM_HALTIUM
 
+config NRF_FORCE_CONSTANT_LATENCY
```

Comment:
There is already a feature like that in nRF54L familiy.
https://github.com/zephyrproject-rtos/zephyr/blob/9ae10ea9b216c609a31e0398c479ae1a701616af/soc/nordic/nrf54l/Kconfig#L79

Can you rename it to SOC_NRF_FORCE_CONSTLAT here and remove the Kconfig from nrf54l/Kconfig.

Reply:
I did not know that existed, and based on comments from @cvinayak the option is probably broken when used with a Bluetooth controller